tests/test_exposure.py

```diff
@@ -777,6 +777,39 @@ def testGetCutout(self):
                     with self.assertRaises(pexExcept.InvalidParameterError, msg=msg):
                         self.smallExposure.getCutout(cutoutCenter, cutoutSize)
 
+    def testGetConvexPolygon(self):
+        """Test the convex polygon."""
+        # Check that we do not have a convex polygon for the plain exposure.
+        self.assertIsNone(self.exposureMiOnly.convex_polygon)
+
+        # Check that all the points in the bounding box are in the polygon
+        bbox = self.exposureMiWcs.getBBox()
+        x, y = np.meshgrid(np.arange(bbox.getBeginX(), bbox.getEndX()),
+                           np.arange(bbox.getBeginY(), bbox.getEndY()))
+        wcs = self.exposureMiWcs.wcs
+        ra, dec = wcs.pixelToSkyArray(x.ravel().astype(np.float64),
+                                      y.ravel().astype(np.float64))
+
+        poly = self.exposureMiWcs.convex_polygon
+        contains = poly.contains(ra, dec)
+
+        self.assertTrue(np.all(contains))
```

Comment:
I wonder if this might provide more useful fail output if it was `assertEqual(contains, np.ones(len(contains), dtype=bool)`? And similarly for the negative version just below.

Reply:
poking this one...

Reply:
Already done, note the outdated badge.  I guess I didn't click an emoji...